config/locales/candidate_mailer.yml

```diff
@@ -86,3 +86,8 @@ en:
         subject: 'Action required: it looks like you submitted a duplicate application'
       multiple_acceptances:
         subject: 'Action required: it looks like you’ve accepted two offers'
+    ucas_match_reminder_email:
+      duplicate_applications:
+        subject: 'Action required: Please withdraw one of your duplicate applications'
```

Comment:
I missed this earlier. Based on this discussion -> https://ukgovernmentdfe.slack.com/archives/CPH8J9G65/p1606217719066500?thread_ts=1606212747.059500&cid=CPH8J9G65 we should use lowercase after the colon.